docs/UserGuide.md

```diff
@@ -197,6 +197,17 @@ Clears all entries from the address book.
 
 Format: `clear`
 
+### Undo last command : `undo`
+
+Revert the last change to the contact list.
+
+Since you can only undo the most recent change, entering `undo` consecutively
+will only show an error message (`No new changes has been made to the data`).
+Likewise, entering `undo` with no prior changes will also display the same error message.
+
+**Note:** `undo` does not affect commands that do not modify the contact list
+(`help`, `list`, `find` etc.). 
+
```

Comment:
Maybe can give examples?

Reply:
I'll merge it first, then for v1.4 I'll look into improving DG with better examples.